back/spec/services/json_forms_ideas_override_spec.rb

```diff
@@ -159,5 +158,12 @@
       expect(ui_schema.dig(:elements)&.find { |e| e[:options][:id] == 'extra' }.dig(:elements)&.count).to eq 1
       expect(ui_schema.dig(:elements)&.any? { |e| e[:options][:id] == 'mainContent' }).to eq true
     end
+
+    it 'all non built-in fields have a nested path' do
```

Comment:
Prefer starting with a verb after `it`, so that it reads like an English sentence.